src/main/java/org/terasology/logic/health/DamageResistEventHandler.java

```diff
@@ -0,0 +1,48 @@
+package org.terasology.logic.health;
+
+import javafx.event.EventHandler;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.EventPriority;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.prefab.Prefab;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.characters.CharacterSystem;
+import org.terasology.logic.health.HealthComponent;
+import org.terasology.logic.health.event.BeforeDamagedEvent;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.sql.SQLOutput;
```

Comment:
Why is this an import? Probably a left over from IDE weirdness I assume?

Reply:
yeah probably :D